app/src/main/res/layout/layout_quick_action_sheet.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/quick_action_sheet"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/off_white">
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/quick_action_sheet_handle"
+        android:layout_marginTop="3dp"
+        android:layout_marginBottom="10dp"
+        android:layout_width="24dp"
+        android:layout_height="2dp"
+        android:layout_gravity="center_horizontal"
+        android:background="@drawable/ic_drawer_pull_tab"/>
+
+    <LinearLayout
+        android:orientation="horizontal"
+        android:layout_gravity="bottom"
+        android:layout_width="match_parent"
+        android:layout_marginBottom="3dp"
+        android:layout_height="wrap_content"
+        android:background="@color/off_white">
+
+        <androidx.appcompat.widget.AppCompatImageButton
```

Comment:
This isn't strictly necessary, but using compound drawables above TextViews would improve performance here, making animations even smoother.

Reply:
Would this allow them to still have onClickListeners when we implement that in the future?

Reply:
Yes.